modules/core/src/test/java/org/apache/ignite/internal/processors/cache/expiry/IgniteCacheExpiryPolicyTestSuite.java

```diff
@@ -57,7 +57,8 @@
 
     IgniteCacheExpireWhileRebalanceTest.class,
 
-    ExpiryPolicyInfoLoggingTest.class
+    ExpiryPolicyInfoLoggingTest.class,
```

Comment:
Is there a test that checks ttl-cleanup-worker for persistence database? I'd propose to enhance GridCacheTtlManagerSelfTest with parameter persistence.

Reply:
Parameter added to the test